inc/html.class.php

```diff
@@ -6548,4 +6573,85 @@ static function getInvertedColor($hexcolor = "", $bw = true, $sbw = true) {
          + str_pad($g, 2, '0', STR_PAD_LEFT)
          + str_pad($b, 2, '0', STR_PAD_LEFT);
    }
+
+   /**
+    * Compile SCSS styleshet
+    *
+    * @param array $args Arguments. May contain:
+    *                      - v: version to append (will default to GLPI_VERSION)
+    *                      - debug: if present, will not use Crunched formatter
+    *                      - file: filerepresentation  to load
+    *                      - reload: force reload and recache
+    *                      - nocache: do not use nor update cache
+    *
+    * @return string
+    */
+   public static function compileScss($args) {
+      global $GLPI_CACHE;
+
+      $ckey = isset($args['v']) ? $args['v'] : GLPI_SCHEMA_VERSION;
+
+      $scss = new Compiler();
+      if ($_SESSION['glpi_use_mode'] != Session::DEBUG_MODE && !isset($args['debug'])) { // mode debug
+         $scss->setFormatter('Leafo\ScssPhp\Formatter\Crunched');
```

Comment:
I think you could always use the `Crunched` formatter. For debug mode, the best to do is to use source maps (`$scss->setSourceMap(Compiler::SOURCE_MAP_INLINE);`)

Reply:
Part of https://github.com/trasher/glpi/compare/feature/scss...cedric-anne:feature/scss?expand=1#diff-298092ebe134c5b9ccbc7042328fb02cR1307 .
Could be done in another PR if you want.

Reply:
I'd prefer, yes ;)